probe_scraper/check_repositories.py

```diff
@@ -0,0 +1,52 @@
+import os
+from pathlib import Path
+
+import requests as reqs
+import yaml
+from glean_parser.lint import lint_yaml_files
+
+GITHUB_RAW_URL = "https://raw.githubusercontent.com"
+REPOSITORIES = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "repositories.yaml"
+)
+validation_errors = []
+with open(REPOSITORIES) as data:
+    repos = yaml.load(data, Loader=yaml.FullLoader)
+    for repo in repos:
+        repo_url = repos[repo]["url"]
+        branch = "master"
+        if "branch" in repos[repo]:
+            branch = repos[repo]["branch"]
+        metrics_files = []
+        if "metrics_files" in repos[repo]:
+            metrics_files = repos[repo]["metrics_files"]
+        if metrics_files:
+            temp_url = (
+                GITHUB_RAW_URL
+                + repo_url.replace("https://github.com", "")
+                + "/"
+                + branch
+                + "/"
+                + metrics_files[0]
+            )
+            print(f"Downloading {repo} metrics_file from {temp_url}...")
```

Comment:
Ideally, this check won't find any problems (that what we'd like to see). So let's remove these print statements and only have output if we've found errors or something problematic (like a network error) happens.  That way, if something goes wrong it's clear what happened, but if everything is ok we aren't filling the output with "noise".

Reply:
ok